vtr_flow/arch/timing/EArch.xml

```diff
@@ -126,7 +126,7 @@
         <!-- data lines can be broken down into smaller bit widths minimum size 1 -->
         <port name="data2" clock="clk"/>
         <!-- data lines can be broken down into smaller bit widths minimum size 1 -->
-        <port name="clk" is_clock="1"/>
+        <port name="clk" is_clock="1" edge="falling"/>
```

Comment:
Why does this memory have a falling edge trigger now?

If you need one for testing purposes, it would be better to make a new architecture with a clear comment.

Reply:
For now I reverted this commit because it is not used anywhere at this point.